src/DockerContainerInstance.php

```diff
@@ -86,6 +86,24 @@ public function execute($command): Process
         return $process;
     }
 
+    /**
+     * @param string|array $command
+     *
+     * @return \Symfony\Component\Process\Process
+     */
+    public function makeProcess($command): Process
```

Comment:
I like the added functionality, but not the implementation. The code of `execute` appears twice now in the class. I think it would be better to add a method `configureProcess` that accepts a callable.

That callable should be kept on `DockerContainerInstance` as an instance variable. In `execute` the created process can be passed to the kept callable.

Could you make this modification?